src/usr/local/www/interfaces_vxlan.php

```diff
@@ -0,0 +1,140 @@
+<?php
+/*
+ * interfaces_vxlan.php
+ *
+ * part of pfSense (https://www.pfsense.org)
+ * Copyright (c) 2004-2013 BSD Perimeter
+ * Copyright (c) 2013-2016 Electric Sheep Fencing
```

Comment:
Did you copy the content from another file?  If not and it's a brand new file, only Rubicon copyright is required